### DISCUSSION TITLE
ImportError: cannot import name 'PlanningContext' from partially initialized module

### DISCUSSION THREAD

**Discussion by aoloo-r** [StanfordVL/BEHAVIOR-1K#1008] [Q&A]
I get this error. How can i resolve it 
from omnigibson.action_primitives.starter_semantic_action_primitives import (
ImportError: cannot import name 'PlanningContext' from partially initialized module 'omnigibson.action_primitives.starter_semantic_action_primitives' (most likely due to a circular import) (/home/Austine/OmniGibson/omnigibson/action_primitives/starter_semantic_action_primitives.py)

**Comment by hang-yin** [comment]
Hi @Roythescientist , do you mind showing me what happens before this line? I can not reproduce this by only running this single line.
